pkg/operator/controller/dns_status.go

```diff
@@ -17,6 +18,9 @@ import (
 	"k8s.io/apimachinery/pkg/util/intstr"
 )
 
+// conditionUpdateInterval is the update interval for progressing and degraded conditions.
+const conditionUpdateInterval = 20 * time.Second
```

Comment:
Should this be slightly longer than the lame duck interval? +2s perhaps?

Reply:
Makes sense. I've made it 22s.